packages/paste-core/layout/stack/src/index.tsx

```diff
@@ -19,7 +19,7 @@ type StackOrientationOptions = 'horizontal' | 'vertical';
 export type StackOrientation = ResponsiveValue<StackOrientationOptions>;
 
 interface StackStyleProps extends Pick<LayoutProps, 'display'>, Pick<FlexboxProps, 'alignItems' | 'flexWrap'> {}
-export interface StackProps extends Pick<BoxProps, 'as'>, React.HTMLAttributes<any> {
+export interface StackProps extends BoxProps {
```

Comment:
Extending `BoxProps` here isn't necessarily true as we `safelySpreadBoxProps`, removing style props passed down. You probably want `BoxElementProps` instead.

Reply:
🧠